### PR TITLE
paddle.mean: add attr axis, keepdim

### PR DESCRIPTION
**Pull request by zhupengyang**
### PR types

Others

### PR changes

APIs

### Describe

- paddle.mean add attr axis, keepdim
- refine doc

image

### CODE REVIEW

**Review by hong19860320 — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -4399,10 +4399,6 @@ def reduce_sum(input, dim=None, keep_dim=False, name=None):
 
 def reduce_mean(input, dim=None, keep_dim=False, name=None):
```
Comment (XiaoguangHu01) [inline python/paddle/fluid/layers/nn.py:4401]: 加一下@deprecated

**Review by XiaoguangHu01 — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (zhupengyang) [reply python/paddle/fluid/layers/nn.py:4401]: 好的，下个PR加

**Review by swtkiwi — DISMISSED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**